pulsar-broker-common/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGeneratorUtils.java

```diff
@@ -59,7 +59,7 @@ public static void generateSystemMetrics(SimpleTextOutputStream stream, String c
             Collector.MetricFamilySamples metricFamily = metricFamilySamples.nextElement();
 
             // Write type of metric
-            stream.write("# TYPE ").write(metricFamily.name).write(' ')
+            stream.write("# TYPE ").write(metricFamily.name + getTypeNameSuffix(metricFamily.type)).write(' ')
```

Comment:
I think we might want to use the following:

```suggestion
            stream.write("# TYPE ").write(metricFamily.name).write(getTypeNameSuffix(metricFamily.type)).write(' ')
```

Reply:
@Shoothzj - do you disagree with this proposed change?

Reply:
@michaeljmarshall Sorry, I missed that, fixed